lib/methods/index.js

```diff
@@ -178,6 +178,7 @@ module.exports = (
     state = state || uuidv4()
     redirectUri = redirectUri || config.redirectUriFqdn
     serviceId = serviceId || config.serviceId
+    clientId = clientId || config.clientId
 
     if (forceLogin === 'yes') {
       forceLogin = true
```

Comment:
does there need to be an "else" ie forceLogin = forceLogin ==='yes' ? true : false

Reply:
I'd be hesitant to change that as part of this pr as it's an established piece of code that's been there since the beginning.
It can maybe be revisited at a later date.